source/Account.cpp

```diff
@@ -136,6 +136,10 @@ string Account::Step(int64_t assets, int64_t salaries, int64_t maintenance)
 	maintenanceDue += maintenance;
 	bool missedPayment = false;
 
+	credits -= escortCosts;
+	if(escortCosts != 0)
+		out << "You paid " << escortCosts << " credits for escorts. ";
```

Comment:
```suggestion
	credits -= escortSalaries;
	if(escortSalaries != 0)
		out << "You paid " << escortSalaries << " credits for escorts. ";
```
If you agree with changing everything to escortSalaries then it needs to be changed here as well